### PR TITLE
Use set of strings for tags instead of one string

### LINKED ISSUE #310: Use set of strings for tags instead of one string

### PR DESCRIPTION
**Pull request by liry**
Fixes #310 (Replaces #311 PR)

### CODE REVIEW

**Comment by jimirocks** [comment]
please fix the typo in 2nd commit `s/Enty/Entry`

File: src/test/java/com/gooddata/util/TagsTestClass.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+package com.gooddata.util;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.util.Set;
+
+public class TagsTestClass {
```
Comment (jimirocks) [inline src/test/java/com/gooddata/util/TagsTestClass.java]: can be package private?

File: src/main/java/com/gooddata/md/ScheduledMail.java [hunk 2]
```diff
@@ -3,14 +3,9 @@
  */
 package com.gooddata.md;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.*;
```
Comment (jimirocks) [inline src/main/java/com/gooddata/md/ScheduledMail.java]: I am not sure if we have any rules about (not)using star imports?

Comment (liry) [inline src/main/java/com/gooddata/md/ScheduledMail.java]: I don't know about any agreement. And there are also usages of star imports in other classes, so lets resolve it later once we have some agreement.

**Comment by liry** [comment]
@jimirocks typo in commit message fixed

**Pull request merged**